.clang-format

```diff
@@ -0,0 +1,39 @@
+# Defines the Chromium style for automatic reformatting.
```

Comment:
I don't think we should just copy the .clang-format file from Chromium, as these demos are not part of Chromium and won't become so. I think it'd be better to have a minimal file (which seems like it'd just be the first 3 lines?).

Reply:
done